app/views/cart_items/_cart_items.html.erb

```diff
@@ -0,0 +1,26 @@
+<tr>
+
+  <td><%= image_tag item.image_url, height: '300', width: '300', id: "#{item.id}_item" %></td>
```

Comment:
In previous commit, I had changed these height and width variables to be 100px each so that more cart items were visible without scrolling.